Add Truthiness/Falsiness with v-bind (#2196)
vuejs/vuejs.org: src/v2/api/index.md

  ```

  `.camel` is not needed if you are using string templates, or compiling with `vue-loader`/`vueify`.
  

>   ## Truthiness/Falsiness with `v-bind`

Reviewer: ```suggestion
  **Note on truthiness and falsiness
```

Reviewer: A level 2 heading here will break the structure of the document/section. I'd suggest using the same style as other list items (Example, See also and so on).

Author: Yes, you right

>   When used with `v-bind`, the only falsy values are now: `null`, `undefined`, and `false`. This means `0` and empty strings will render as truthy. So for example, `v-bind:draggable="''"` will render as `draggable="true"`.

Reviewer: ```suggestion
  When used with `v-bind`, the only falsy values are `null`, `undefined`, and `false`. This means `0` and empty strings will be rendered as truthy. For example, `v-bind:draggable="''"` will become `draggable="true"`.
```

>   For enumerated attributes, in addition to the falsy values above, the string `"false"` will also render as `attr="false"`.

Reviewer: ```suggestion
  For enumerated attributes, in addition to the falsy values above, the string `'false'` will also be rendered as `attr="false"`.
```

Reviewer: Can we have an example to make this statement clearer? I'm not sure a beginner will get what "enumerated attributes" are.

Author: added
